prow/istio-pilot-multicluster-e2e.sh

```diff
@@ -23,4 +23,4 @@ CLUSTERREG_DIR="${CLUSTERREG_DIR:-$(mktemp -d /tmp/clusterregXXX)}"
 export CLUSTERREG_DIR
 
 #echo 'Running pilot multi-cluster e2e tests (v1alpha1, noauth)'
-./prow/e2e-suite.sh --timeout 50 --cluster_registry_dir="$CLUSTERREG_DIR" --single_test e2e_pilotv2_v1alpha3 "$@"
+./prow/e2e-suite.sh --timeout 90m --cluster_registry_dir="$CLUSTERREG_DIR" --single_test e2e_pilotv2_v1alpha3 "$@"
```

Comment:
This looks really long, are you sure that this is best approach?

Reply:
I'm not sure, but it times out often.. The multicluster one times out 190 times out of 323 PRs

```
     34 e2e-bookInfoTests-envoyv2-v1alpha3
      1 e2e-dashboard
     34 e2e-mixer-no_auth
      6 e2e-mixer-no_auth-mcp
     10 istio_auth_sds_e2e
     38 istio-integ-k8s-tests
     46 istio-pilot-e2e-envoyv2-v1alpha3
     11 istio-pilot-e2e-envoyv2-v1alpha3-mcp
    190 istio-pilot-multicluster-e2e
      4 istio-unit-tests
```

Maybe it times out less now, I am still downloading the test results, so these results may be old

Reply:
If you look here it takes a long time too https://prow.istio.io/?job=istio-pilot-multicluster-e2e-master

Reply:
My rationale here was that a test should never time out while it is still running. Perhaps if we were in a better state where a test failure was alarming and we went in and reworked tests to make them faster if they failed due to timeouts it would make sense to have hard limits on the tests, but right now it just causes flakes I think

Reply:
Ok, let's increase the timeout, but it should be investigated why it takes so long .